tests/fixtures/config/env/test.yml

```diff
@@ -67,8 +67,10 @@ engines:
           - name: Plugin 3 of 3 (#2 should be HIDDEN)
             hook: "{self}/publish_file.py:{config}/visible_plugin3.py"
             settings: {}
-
-            
+          - name: Vitor's Plugin
```

Comment:
who is vitor?

Reply:
https://github.com/shotgunsoftware/tk-multi-publish2/pull/79/files#diff-f12f0df5921cb95cb68e97cf4ea2df9aR21 :)